packages/common/core-interfaces/src/test/types/validateCoreInterfacesPrevious.generated.ts

```diff
@@ -8,38 +8,12 @@
  * Generated by flub generate:typetests in @fluid-tools/build-cli.
  */
 
+import type { TypeOnly, MinimalType, FullType } from "@fluidframework/build-tools";
 import type * as old from "@fluidframework/core-interfaces-previous/internal";
 
 import type * as current from "../../index.js";
 
-type ValueOf<T> = T[keyof T];
-type OnlySymbols<T> = T extends symbol ? T : never;
-type WellKnownSymbols = OnlySymbols<ValueOf<typeof Symbol>>;
-/**
- * Omit (replace with never) a key if it is a custom symbol,
- * not just symbol or a well known symbol from the global Symbol.
- */
-type SkipUniqueSymbols<Key> = symbol extends Key
-	? Key // Key is symbol or a generalization of symbol, so leave it as is.
-	: Key extends symbol
-		? Key extends WellKnownSymbols
-			? Key // Key is a well known symbol from the global Symbol object. These are shared between packages, so they are fine and kept as is.
-			: never // Key is most likely some specialized symbol, typically a unique symbol. These break type comparisons so are removed by replacing them with never.
-		: Key; // Key is not a symbol (for example its a string or number), so leave it as is.
-/**
- * Remove details of T which are incompatible with type testing while keeping as much as is practical.
- *
- * See 'build-tools/packages/build-tools/src/typeValidator/compatibility.ts' for more information.
- */
-type TypeOnly<T> = T extends number
-	? number
-	: T extends boolean | bigint | string
-		? T
-		: T extends symbol
-			? SkipUniqueSymbols<T>
-			: {
-					[P in keyof T as SkipUniqueSymbols<P>]: TypeOnly<T[P]>;
-				};
+declare type MakeUnusedImportErrorsGoAway<T> = TypeOnly<T> | MinimalType<T> | FullType<T> | typeof old | typeof current;
```

Comment:
Can we have a brief callout in the PR description about what improvements to type tests are brought in by build-tools 0.39?

Reply:
@CraigMacomber / @tylerbutler, can you add such a note?

Reply:
Changes are from https://github.com/microsoft/FluidFramework/pull/19224, I'll add a note

Reply:
done.